(DOCSP-6610): Adding Markup Guide to Style Guide. (#21)
mongodb/docs-meta: source/style-guide/markup/format/lists.txt

=====
Lists
=====

.. default-domain:: mongodb

.. include:: /includes/styles/corrections.rst

Each level of a list must start and end with a blank line.

Unordered Lists
~~~~~~~~~~~~~~~

- Use hyphens (``-``) to indicate items in an unordered list. 

> - Use the hyphen at every level of a multi-level unordered list.

Reviewer: I think this can be removed since it duplicates information presented in the second sentence of the following item.

- To create a multi-level list, indent each level two spaces for each
  subordinate level. Each subordinate level must start and end with a
  blank line.

.. list-table::
   :header-rows: 1
   :widths: 50 50

   * - Use
     - Avoid

   * -

       .. code-block:: rst

          - List item 1

            - List item 1a
            - List item 1b

          - List item 2

            - List item 2a
            - List item 2b

     -

       .. code-block:: rst
          :copyable: false

          * List item 1
            - List item 1a
            - List item 1b

          * List item 2
            - List item 2a
            - List item 2b

Ordered Lists
~~~~~~~~~~~~~

- Use a number followed by a period for the first list item (``1.``,

>   ``a.``, etc.). This informs the parser which numbering scheme you are

Reviewer 2: ```suggestion
  ``a.``, and so on). This informs the parser which numbering scheme you are
```

  using for this level of the list.
- Use a hash sign followed by a period (``#.``) for each subsequent
  list item. The parser replaces the hash with the next number in the
  sequence.
- To create a multi-level list, indent each level three spaces for each
  subordinate level. Each subordinate level must start and end with a
  blank line.

.. list-table::
   :header-rows: 1
   :widths: 50 50

   * - Use
     - Avoid

   * -

       .. code-block:: rst

          1. List item 1

            a. List item 1a
            #. List item 1b

          #. List item 2

            a. List item 2a
            #. List item 2b

     -

       .. code-block:: rst
          :copyable: false

          1. List item 1
             a. List item 1a
             b. List item 1b
          2. List item 2
             a. List item 2a
             b. List item 2b

> Definition Lists

Reviewer: It wasn't clear to me what these are based on the bullet point. Could benefit from a description.

~~~~~~~~~~~~~~~~

> - Do not use definition lists instead of subheadings. You can link to

Reviewer 2: ```suggestion
- Don't use definition lists instead of subheadings. You can link to
```

>   any headings, but not to a term in a definition list.

Reviewer 2: Is this true?
